addons/mail/static/tests/mock_server/old_mock_models/discuss_channel_rtc_session.js

```diff
@@ -1,7 +1,9 @@
 /** @odoo-module alias=@mail/../tests/helpers/mock_server/models/discuss_channel_rtc_session default=false */
 
-import { patch } from "@web/core/utils/patch";
-import { MockServer } from "@web/../tests/helpers/mock_server";
+// import { patch } from "@web/core/utils/patch";
+// import { MockServer } from "@web/../tests/helpers/mock_server";
+var patch = () => {};
+var MockServer = { prototype: {} };
```

Comment:
there are a lot of strange intermediate steps in this specific move commit `[MOV] mail: dischoot (5) `